### ISSUE TITLE
Relationship Object Id's error

### ISSUE THREAD

**Issue by sfielder** [balderdashy/sails-mongo#205]
Where am I going wrong? Just loaded up the latest npm install sails-mongo --save into my project and the fun still exists "relatedObject_id": { "_bsontype": "ObjectID", "id": "T^$0bb\u0000\u0002\u0003\u0000\u0000" }

Thanks

"name": "sails-mongo",
 "version": "0.10.4",

**Comment by listepo** [comment]
+1

**Comment by sailsbot** [comment]
Thanks for posting, @sfielder.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
